src/logic/operate.js

```diff
@@ -0,0 +1,21 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const num1 = Big(numberOne);
+  const num2 = Big(numberTwo);
+  let result = 0;
+
+  const operations = {
+    '+': () => num1.plus(num2),
+    '-': () => num1.minus(num2),
+    x: () => num1.times(num2),
```

Comment:
[Optional]
You have named the multiplication symbol as 'X'. So, I think you are gonna call this function instead of x: to 'X':. Isn't it?